### PR TITLE
Darcli theme

### PR DESCRIPTION
**Pull request by gjuniioor**
Added [darcli](https://github.com/gjuniioor/darcli) theme.

### CODE REVIEW

**Comment by gynter** [comment]
Thanks for the PR! Thumbnail image aspect ratio is wrong, the image is stretched out. Please correct the image.

File: _posts/2016-05-02-darcli.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Darcli
+date: 2016-05-02 17:00:00
+homepage: https://github.com/gjuniioor/darcli
+download: https://github.com/gjuniioor/darcli/archive/master.zip
+demo: https://gjuniioor.github.io/darcli/
+author: Gildásio Júnior aka gjuniioor
+thumbnail: darcli.png
+license: GPL3
```
Comment (gynter) [inline _posts/2016-05-02-darcli.markdown]: To keep up with similar formatting, please use `GNU General Public License Version 3` as license.

**Comment by gjuniioor** [comment]
Oh, sorry.

Look now, @gynter . It's everything okay? :D

**Comment by gynter** [comment]
Nice, thanks. Merged!

**Pull request merged**